### PR TITLE
fix(install): let pathes in 'env_add_path' be added ascendantly

### PR DESCRIPTION
**Pull request by niheaven**
Fix https://github.com/lukesampson/scoop/issues/3785

### CODE REVIEW

File: lib/install.ps1 [hunk 1]
```diff
@@ -938,6 +938,7 @@ function find_dir_or_subdir($path, $dir) {
 
 function env_add_path($manifest, $dir, $global, $arch) {
     $env_add_path = arch_specific 'env_add_path' $manifest $arch
+    $env_add_path = $env_add_path[-1..-$env_add_path.Length]
```
Comment (h404bi) [inline lib/install.ps1]:
```suggestion
    # GH-3785: Add path in ascending order.
    $env_add_path = $env_add_path[-1..-$env_add_path.Length]
```

File: lib/install.ps1 [hunk 2]
```diff
@@ -938,6 +938,8 @@ function find_dir_or_subdir($path, $dir) {
 
 function env_add_path($manifest, $dir, $global, $arch) {
     $env_add_path = arch_specific 'env_add_path' $manifest $arch
+    # GH-3785: Add path in ascending order.
+    $env_add_path = $env_add_path[-1..-$env_add_path.Length]
```
Comment (Ash258) [inline lib/install.ps1]: Would not be more cleaner to use `[Array]::Reverse($env_add_path)`?

**Comment by Ash258** [comment]
Is there any possibility of breaking current behaviour? I cannot think about anything right now.

**Comment by niheaven** [comment]
Just order in %PATH%, which shouldn't break anything. Maintainer typically write `env_add_path` ascendantly, I thought.

**Comment by h404bi** [comment]
ping @r15ch13

**Pull request merged**